File: Bindings~/java/src/main/java/com/alttester/Notifications/AltLogNotificationResultParams.java
package com.alttester.Notifications;

public class AltLogNotificationResultParams {

    public String message;
    public String stackTrace;
    public int level;

    public AltLogNotificationResultParams(String message, String stackTrace, int level) {
        this.message = message;
        this.stackTrace = stackTrace;
        this.level = level;
    }
}

File: Assets/AltTester/Runtime/AltDriver/Notifications/AltLogNotificationResultParams.cs
/*
    Copyright(C) 2026 Altom Consulting
*/

using AltTester.AltTesterSDK.Driver.Logging;

namespace AltTester.AltTesterSDK.Driver.Notifications
{
    public class AltLogNotificationResultParams
    {
        public string message;
        public string stackTrace;
        public AltLogLevel level;

        public AltLogNotificationResultParams(string message, string stackTrace, AltLogLevel level)
        {
            this.message = message;
            this.stackTrace = stackTrace;
            this.level = level;
        }
    }
}
